rogue-thi-app/pages/calendar.js

```diff
@@ -38,7 +38,7 @@ export const calendar = rawCalendar.map(x => ({
 
 export async function loadExamList () {
   const examList = await API.getExams()
-  return examList
+  let tempList = examList
```

Comment:
as `filter` returns a new list and `sort` returns the list itself too we do not really need a mutable variable here. A common thing to do is to simply chain the calls as in `return examList.map(...).filter(...).sort(...)`

I guess i can implement that in a new commit. Thanks for the PR :smiley: